docs/modules/deployment/pages/time-series-storage/newts/newts.adoc

```diff
@@ -9,65 +9,80 @@ This section describes how to configure your {page-component-title} instance to
 
 Follow these steps to set up Newts on your {page-component-title} instance:
 
-. Create a configuration file with your time series database settings:
-+
+.Create a configuration file with the time series storage settings
 [source, console]
+----
 sudo vi etc/opennms.properties.d/timeseries.properties
+----
 
-. Configure the storage strategy:
-+
+.Configure Newts as the time series strategy
 [source, properties]
 ----
-org.opennms.rrd.storeByForeignSource=true <1>
-org.opennms.timeseries.strategy=newts <2>
+# Configure storage strategy
+org.opennms.rrd.storeByForeignSource=true<1>
+org.opennms.timeseries.strategy=newts<2>
+
+# One year in seconds
+org.opennms.newts.config.ttl=31540000<3>
+
+# Seven days in seconds
+org.opennms.newts.config.resource_shard=604800<4>
+
+# Configure Newts time series storage connection
+org.opennms.newts.config.driver_settings_file=/opt/opennms/etc/cassandra-driver.conf<5>
 ----
-<1> Associate time series data using the foreign source and ID instead of the database-generated node ID.
-<2> Set time series strategy to use Newts.
-If you are enabling the xref:time-series-storage/timeseries/time-series-storage.adoc#ga-dual-write-newts[dual write plugin] on an existing {page-component-title} installation and you want to keep historical metrics, make sure that the written data has expired before you set `org.opennms.timeseries.strategy` to `newts`.
 
-. Configure the Newts time series storage connection:
-+
-[source, properties]
+<1> Associate time series data by the foreign ID instead of the database-generated Node-ID.
+<2> Set time series strategy to use `newts`.
+<3> Retention rate for the time series data.
+<4> Shard metrics every 7 days.
+<5> The path to your driver configuration file
+
+.Create a configuration file with the time series storage settings
+[source, console]
 ----
-org.opennms.newts.config.hostname=cassandra-ip1,cassandra-ip2 <1>
-org.opennms.newts.config.keyspace=newts <2>
-org.opennms.newts.config.port=9042 <3>
+sudo vi etc/cassandra-driver.conf
 ----
-<1> Host or IP addresses of the Cassandra cluster nodes.
-Can be a comma-separated list.
-<2> Name of the keyspace which is initialized and used.
-<3> Port to connect to Cassandra.
 
-. Set the retention rate and shard rate:
-+
+. Configure the Cassandra driver settings
 [source, properties]
 ----
-# One year in seconds
-org.opennms.newts.config.ttl=31540000 <1>
-
-# Seven days in seconds
-org.opennms.newts.config.resource_shard=604800 <2>
+datastax-java-driver {
+  basic.contact-points = [ "example-node1:9042", "example-node2:9042" ]<1>
+  session-keyspace = "newts"<2>
+  basic.load-balancing-policy {
+    local-datacenter = datacenter1<3>
+  }
+  advanced.auth-provider {
+    class = PlainTextAuthProvider <4>
```

Comment:
Fine but might be good to put something to encourage use of encryption here if possible.